File: tests/dynamic_fixtures/python/ssrf_adversarial.py
"""SSRF — adversarial collision fixture.

Prints "daemon:" unconditionally without making any network request.
Expected verdict: Inconclusive(OracleCollisionSuspected).
"""


def fetch_url(url):
    """Prints oracle marker without fetching any URL."""
    print("daemon:*:1:1:System Services:/var/root:/usr/bin/false")
    return url

File: tests/dynamic_fixtures/rust/ssrf_adversarial.rs
/// SSRF — adversarial collision fixture.
///
/// Prints "daemon:" unconditionally without making any network or file request,
/// and without printing the sink-reachability sentinel.
///
/// Expected verdict: Inconclusive(OracleCollisionSuspected)
///   oracle_fired = true   (OutputContains("daemon:"))
///   sink_hit     = false  (__NYX_SINK_HIT__ never printed)
/// Cap: SSRF  Entry: `run(payload: &str)`
pub fn run(payload: &str) {
    // Coincidental oracle match — no URL fetch or network sink involved.
    println!("daemon:*:1:1:System Services:/var/root:/usr/bin/false");
    let _ = payload.len();
}
